src/components/new-request/search.vue

```diff
@@ -44,40 +43,49 @@
         </v-select>
       </v-col>
 
-      <!-- location (aka jurisdiction) -->
-      <v-col cols="12" md="4" lg="4" :class="{'px-3': !isMobile }">
-        <v-tooltip id="location-options-select"
-                   top
-                   content-class="top-tooltip"
-                   transition="fade-transition"
-                   :disabled="!location || location === 'BC' || isMobile">
+      <!-- display a dummy input box here when Jurisdiction and Entity Type are not shown -->
+      <v-col v-if="!showJurisdiction && !showEntityType" cols="12" md="6" class="py-0">
+        <v-text-field filled disabled />
+      </v-col>
+
+      <!-- Jurisdiction -->
+      <v-col v-if="showJurisdiction" cols="12" md="6" class="py-0">
+        <v-tooltip
+          id="location-options-select"
+          top
+          content-class="top-tooltip"
+          transition="fade-transition"
+          :disabled="!getLocation || getLocation === 'BC' || isMobile"
+        >
           <template v-slot:activator="scope">
             <div v-on="scope.on">
-              <v-select :error-messages="getErrors.includes('location') ? 'Please select a jurisdiction' : ''"
-                        :items="getLocationOptions"
-                        :disabled="locationDisabled"
```

Comment:
I renamed locationDisabled -> showJurisdiction and used it to conditionally (v-if) this entire column instead of disabling this menu.

Reply:
clean and self-explanatory name :)